### DISCUSSION TITLE
Any other mechanisms to save gpu memory other than paged attention?

### DISCUSSION THREAD

**Discussion by lyz-sys** [vllm-project/vllm#290] [Q&A]
Hi guys, are there other mechanisms implemented in vllm to save gpu memory other than paged attention? Thank you.

**Comment by zhuohan123** [comment]
Another technique is continuous batching, which reduces padding memory and computation. You can read [this blog](https://www.anyscale.com/blog/continuous-batching-llm-inference) to learn more.
